routes/v0/cases.js

```diff
@@ -20,7 +39,11 @@ casesRouter.post('/', async (req, res) => {
   res.json(createdCase);
 });
 
-casesRouter.put('/:id', async (req, res) => {
+/**
+ * We use asyncHandler(fn) here because expressJS expects the middleware to be a synchronous function,
+ * but editCasePermissions(args) is asynchronous
```

Comment:
If we start having performance issues by needing to do another database fetch, we could cache this.  We would need to be careful about invalidating the cache if this or another server instance changed the case owner, though.